form.yml

```diff
@@ -17,24 +17,21 @@ attributes:
     widget: select
     label: "Node type"
     help: |
-      - **any** - (*12 cores*) Use any available Oakley node. This reduces the
+      - **any** - (*28 cores*) Use any available Owens node. This reduces the
         wait time as there are no node requirements.
-      - **vis** - (*12 cores*) Use an Oakley node that has an NVIDIA Tesla
-        M2070 GPU with hardware rendering enabled for 3D visualization. There
-        are 128 of these nodes on Oakley.
-      - **bigmem** - (*12 cores*) Use an Oakley node that has 192GB of
-        available RAM. There are 8 of these nodes on Oakley.
-      - **hugemem** - (*32 cores*) Use an Oakley node that has 1TB of available
-        RAM as well as 32 cores. There are 1 of these nodes on Oakley.
+      - **gpu** - (*28 cores*) Use an Owens node that has an NVIDIA P100 "Pascal" GPU with hardware rendering enabled for 3D visualization. There are 160 of these nodes on Owens.
```

Comment:
Why did you remove `:vis` support? Does it successfully launch VirtualGL without it on Owens?

Reply:
You can also use the same help text we use for the Abaqus app:

https://github.com/OSC/bc_osc_abaqus/blob/27053997ef29d438349787acd3d0ea0d9ead9ce7/form.yml#L18-L21

Reply:
There are some fixes that have been added in the https://github.com/OSC/bc_osc_ansys_workbench/tree/owens18_PMIU0149 branch. Those will need to be backported to here.